### DISCUSSION TITLE
What is the .todo file?

### DISCUSSION THREAD

**Discussion by TechySC23** [Satanarious/TranslucentFlyoutsConfig#61] [Q&A] [answered]
How did you write the code? It isn't clear. How did you format that? @Satanarious ?

**Comment by Satanarious** [comment, marked as answer]
The .todo file is for a vscode extension called [todo+](https://marketplace.visualstudio.com/items?itemName=fabiospampinato.vscode-todo-plus). Right now, it contains the process to add a new parameter. I maintain it locally and it was not supposed to be version controlled, but I thought the process of adding a new value involves a lot of steps so I added it to the list for future references.

 Can you be more specific? Do you want to know the flow of the code when you say, "How did you write the code?"

**Comment by TechySC23** [comment]
Oh ok. I meant the format of the code. Like how to write the code of that .todo file. I'll check the extension soon!

**Reply by Satanarious** [reply]
Don't forget to mark the above response as resolved if it answered your query correctly.
